docs/nodes/scene/evolver.rst

```diff
@@ -0,0 +1,58 @@
+Evolver
+=======
+
+This node implements a Genetics Algorithm system inspired in Galapagos (the Grasshopper plugin) and the 9th chapter of "The Nature of Code" by Daniel Shiffman https://natureofcode.com/book/chapter-9-the-evolution-of-code/
+
+The system creates a starting population, evaluates the fitness of each "agent" and creates a new population bases on the crossover of the fitter agents and a mutation chance. The new population is evaluated and mixed every iteration rising the global fitness of the population by generating fitter agents.
+
+Every agent is composed by a set of genes that can oscillate among determined values, the fitter agents have more chances to spread their genetic information.
+
+In Sverchok every gen is defined by a "A Number" node. This node type has a minimum and a maximum that will control the variation possibilities of this gen. The node can use all the "A number" nodes of the node-tree or only some of them if they are inside a "Frame Node" that can be selected from the "Genotype" dropdown menu.
+
+The fitness of every member of the population will be evaluated by running the node-tree with the genes of that member and recording the value that is inputted in the "Fitness" socket.
+
+Parameters
+----------
+
+**Genotype**: dropdown menu to chose to use as genes either all 'A Number' nodes or only the ones inside a 'Frame' Node
+
+**Mode**:  "Maximum" and "Minimum" with the maximum selected a higher fitness value is considered better with the minimum a lower fitness will be considered as more fit.
```

Comment:
also, why not to establish threshold for stop with optimal value and with optimal fitness value in enum with minimum and maximum. That will give us possibility to break before last iteration and will save time

Reply:
Done!